template/src/lib/ui/stories/common/Dialog.stories.js

```diff
@@ -83,4 +86,4 @@ export const Default = Template.bind({});
 Default.args = { content: <TextContent /> };
 
 export const Form = Template.bind({});
-Form.args = { content: <FormContent /> };
+Form.args = { content: <FormContent />, maxWidth: false, contentStyle: { width: 383 } };
```

Comment:
Why having to set styles manually?

Reply:
Asked by design  for this specific example

Reply:
Better define widths on `paperWidthXs`,... See https://material-ui.com/api/dialog/